src/containers/profile/Profile.js

```diff
@@ -16,6 +16,14 @@ export class UnconnectedProfile extends Component {
     getUserArticles(userId);
   }
 
+  handleArticleClick = (e, slug) => {
+    const { history, getArticle } = this.props; // eslint-disable-line
+    e.preventDefault();
+    const urlTo = `/articles/${slug}`;
+    getArticle(slug);
+    history.push(urlTo);
```

Comment:
This `getArticle(slug)` should be successful for this  `history.push(urlTo)` to happen. Your logic does not cater for that.

Reply:
We made this fix in @jonathanmusila 's PR 
I don't think I have to redo it here